### PR TITLE
Profile-Component

### LINKED ISSUE #29: Profile Component
As per [Figma](https://www.figma.com/file/qOoG6pKbWKi2ldzHT2cHA6/Crypto-Component---Light-Mode-(Part-2)-(Community)?node-id=1%3A6094&t=zjwAxrDCZeGbfalZ-4)

### Preview

<div align="center">

Image

</div>

### Variants
- [x] Light

### PR DESCRIPTION
**Pull request by Demandtech**
Fix #29 

screencapture-localhost-3000-profile-2022-12-10-01_00_07

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -68,6 +69,10 @@ const router = createBrowserRouter([
         path: 'stories/cardpayment',
         element: <CardPayment />,
       },
+      {
+        path: 'profile',
```
Comment (mrxshinji) [inline src/index.js:73]: path : /stories/profile

**Review by mrxshinji — CHANGES_REQUESTED** [review]
# TODO
- Add a Link component in story-page.js to navigate onto this component.

File: src/routes/profile/profile.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.profile-wrapper {
+ max-width: 360px;
+ margin: 0 auto;
+ display: flex;
+ align-items: center;
+ background: #ffffff;
```
Comment (mrxshinji) [inline src/routes/profile/profile.css:6]: Do you have prettier installed?

indent required for inner part, if you have prettier installed, alt-shift-f will fix this

**Comment by Demandtech** [comment]
Okay I'm on it.

**Review by mrxshinji — APPROVED** [review]
PR approved, thank you

**Review by aftabrehan — COMMENTED** [review]
Excellent PR, @Demandtech 💪 🚀 💯

**Pull request merged**